packages/block/src/block.ts

```diff
@@ -603,17 +644,19 @@ export class Block {
 
   /**
    * Validates the uncle's hash.
+   * @returns true if the uncle's hash is valid, false otherwise.
    */
-  validateUnclesHash(): boolean {
+  uncleHashIsValid(): boolean {
     const uncles = this.uncleHeaders.map((uh) => uh.raw())
     const raw = RLP.encode(uncles)
     return equalsBytes(keccak256(raw), this.header.uncleHash)
   }
 
   /**
    * Validates the withdrawal root
+   * @returns true if the withdrawals trie root is valid, false otherwise
    */
-  async validateWithdrawalsTrie(): Promise<boolean> {
+  async withdrawalsTrieIsValid(): Promise<boolean> {
     if (!this._common.isActivatedEIP(4895)) {
       throw new Error('EIP 4895 is not activated')
```

Comment:
This throws? Shouldn't this `return false`?

Reply:
```typescript
  /**
   * Validates the withdrawal root
   * @returns true if the withdrawals trie root is valid, false otherwise
   */
  async withdrawalsTrieIsValid(): Promise<boolean> {
    if (!this._common.isActivatedEIP(4895)) {
      throw new Error('EIP 4895 is not activated')
```
     
The (new) docs say otherwise :thinking:

Reply:
Hmm, wait, it makes no sense if you call the method if the EIP is activated so I guess it is correct

Reply:
Yeah exactly, makes sense that it throws if 4895 isn't activated as this basically is a misuse of the method.

Reply:
Definitely, will close this thread